css/style.css

```diff
@@ -550,106 +550,186 @@ h5 {
 }
 
 /* Testimonials Section */
-.testimonials {
-  padding: 50px 0px;
-  background-color: rgb(1, 36, 49);
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-flow: wrap;
-  margin: 0;
-}
 .testimonials_heading {
-  padding: 20px 0px;
-  text-align: center;
-  font-weight: bolder;
-  color: white;
-  width: 100%;
+    padding: 20px 0px;
+    text-align: center;
+    font-weight: bolder;
+    color: rgb(11, 10, 10);
+    width: 100%;
 }
+
 .testimonials_heading h1 {
-  font-size: 50px;
+    font-size: 50px;
 }
 
 figure.testimonal-Card {
-  font-family: "Raleway", Arial, sans-serif;
-  position: relative;
-  overflow: hidden;
-  margin: 10px;
-  min-width: 220px;
-  max-width: 310px;
-  width: 100%;
-  color: #333;
-  text-align: left;
-  box-shadow: 0 0 5px rgba(0, 0, 0, 0.15);
-  border-radius: 8px;
+    font-family: "Raleway", Arial, sans-serif;
+    position: relative;
+    overflow: hidden;
+    margin: 0 auto;
+    min-width: 220px;
+    max-width: 310px;
+    width: 100%;
+    color: #333;
+    text-align: left;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.15);
+    border-radius: 8px;
 }
+
 figure.testimonal-Card * {
-  -webkit-box-sizing: border-box;
-  box-sizing: border-box;
-  -webkit-transition: all 0.35s cubic-bezier(0.25, 0.5, 0.5, 0.9);
-  transition: all 0.35s cubic-bezier(0.25, 0.5, 0.5, 0.9);
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    -webkit-transition: all 0.35s cubic-bezier(0.25, 0.5, 0.5, 0.9);
+    transition: all 0.35s cubic-bezier(0.25, 0.5, 0.5, 0.9);
 }
+
 figure.testimonal-Card img {
-  max-width: 100%;
-  vertical-align: middle;
+    max-width: 100%;
+    vertical-align: middle;
 }
+
 figure.testimonal-Card blockquote {
-  position: relative;
-  background-color: #ffffff;
-  padding: 25px 50px 25px 50px;
-  font-size: 0.8em;
-  font-weight: 500;
-  text-align: left;
-  margin: 0;
-  line-height: 1.6em;
-  font-style: italic;
+    position: relative;
+    background-color: #ffffff;
+    padding: 25px 50px 25px 50px;

```

Comment:
If you see that top/bottom are same and left/right are same, You can write like this:

```css
/* padding: top/bottom left/right */
padding: 25px 50px;
````

Conclusion:
```diff
- padding: 25px 50px 25px 50px;
+ padding: 25px 50px;
```